.github/workflows/check.yml

```diff
@@ -0,0 +1,30 @@
+name: Test suite
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.gitlab.com/islandoftex/images/texlive:latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Install PGF
+      run: |
+        tlmgr init-usertree --usertree $PWD
+        echo "TEXMFHOME=$PWD" >> $GITHUB_ENV
+
+    - name: Generate the revision file
+      run: |
+        export GIT_TAG=$(git describe --abbrev=0 --tags)
+        echo "GIT_TAG=$GIT_TAG" >> $GITHUB_ENV
+        l3build tag "$GIT_TAG"
+        cat tex/generic/pgf/pgf.revision.tex
```

Comment:
Not sure this is needed

Reply:
This is just for debugging to see whether the revision file has been generated correctly.